.cargo/config.toml

```diff
@@ -3,6 +3,7 @@ uitest = "test --test compile-test"
 dev = "run --package clippy_dev --bin clippy_dev --manifest-path clippy_dev/Cargo.toml --"
 lintcheck = "run --package lintcheck --bin lintcheck --manifest-path lintcheck/Cargo.toml  -- "
 collect-metadata = "test --test dogfood --features internal -- run_metadata_collection_lint --ignored"
+dogfood = "test --test dogfood"
```

Comment:
We have `dev dogfood` with `--fix` support. So I don't think we need another alias.

Reply:
Somehow I missed that. I've been using `test` directly this entire time.

Reply:
I also only noticed it 2 weeks ago or so :smile: